<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="stacked_w">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="main_menu">
      <widget class="QPushButton" name="new_btn">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>170</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Juego Nuevo</string>
       </property>
      </widget>
      <widget class="QPushButton" name="load_btn">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>260</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Cargar Partida</string>
       </property>
      </widget>
      <widget class="QPushButton" name="exit_btn">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>360</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="new_log">
      <widget class="QPushButton" name="return_new_btn">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>390</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Atrás</string>
       </property>
      </widget>
      <widget class="QPushButton" name="register_btn">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>340</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>¡Comenzar!</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="pass_cbox">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>270</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ver contraseña</string>
       </property>
      </widget>
      <widget class="QLabel" name="info_lb">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>331</width>
         <height>131</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Información</string>
       </property>
      </widget>
      <widget class="QPushButton" name="mode_btn">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>280</y>
         <width>161</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Un solo jugador</string>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>170</y>
         <width>330</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="id_lb">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;  Usuasrio:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pass_lb">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Contraseña:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pass2_lb">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Repite la Contraseña:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="id_line">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pass_line">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="pass2_line">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="log">
      <widget class="QPushButton" name="return_log_btn">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>210</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Atrás</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>290</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>LOG</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="game">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QGraphicsView" name="game_gv"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
